test/bioclip/conftest.py

```
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+import pytest
+
+from camvidlog2.bioclip.ai import get_video_embeddings
+from camvidlog2.bioclip.data import create
+from camvidlog2.vid import generate_frames_cv2
+
+
+@pytest.fixture(name="video_embeddings_path", scope="session")
+def fixture_video_embeddings_path(video_path: Path) -> Path:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)